### PR TITLE
12.0 mig l10n ch pain credit transfer

### PR DESCRIPTION
**Pull request by i-vyshnevska**
base_bank updates

### CODE REVIEW

File: l10n_ch_base_bank/tests/test_create_invoice.py [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
-from odoo.tests import common
+from .base_mixin import BaseInitInvoice
 from odoo import exceptions
 
 
-class TestCreateInvoice(common.TransactionCase):
+class TestCreateInvoice(BaseInitInvoice):
+
+    def setUp(self):
+        super().setUp()
```
Comment (asaunier) [inline l10n_ch_base_bank/tests/test_create_invoice.py]: Do you really need to override this method if it does nothing else than the parent one?

File: l10n_ch_base_bank/tests/base_mixin.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Copyright 2012-2019 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import SavepointCase
+
+
+class BaseInitInvoice(SavepointCase):
+
+    def setUp(self):
+        super(BaseInitInvoice, self).setUp()
```
Comment (asaunier) [inline l10n_ch_base_bank/tests/base_mixin.py]:
```suggestion
        super().setUp()
```

Comment (asaunier) [reply l10n_ch_base_bank/tests/base_mixin.py]: Arguments are not mandatory in super() with Python 3 :)

**Comment by i-vyshnevska** [comment]
can't push directly or even merge this one PR
as git says
'Only those with write access to this repository can merge pull requests.'

**Comment by i-vyshnevska** [comment]
:smile:, sorry

**Pull request merged**